### DISCUSSION TITLE
beginner needs help

### DISCUSSION THREAD

**Discussion by jingxu2021** [rabbitmq/ra#372] [General]
Hi,
I'm a beginner to ra. If my question is naive please forgive me.
Is there a way to show a cluster's status, including leader, current term number, current term start/end time, followers, if a follower is synced or not, if not synced what's the progress, current log id, history snapshots, etc. ? This will be very much helpful. 
Thanks in advance.
Jing

**Comment by kjnilsson** [comment]
There is a new api ra:member_overview/1 that contains a summary map with a
lot of that information. You may have enough just getting the overview from
the leader but you may want to use ra:member/1 to first find the leader
then call it for each of the followers as well.

On Thu, 11 May 2023 at 19:50, jingxu2021 ***@***.***> wrote:

> Hi,
> I'm a beginner to ra. If my question is naive please forgive me.
> Is there a way to show a cluster's status, including leader, current term
> number, current term start/end time, followers, if a follower is synced or
> not, if not synced what's the progress, current log id, history snapshots,
> etc. ? This will be very much helpful.
> Thanks in advance.
> Jing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rabbitmq/ra/discussions/372>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJAHFG7KJIHFO5Y7IBNYZTXFUYI5ANCNFSM6AAAAAAX6Q5FGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Karl Nilsson*

**Comment by jingxu2021** [comment]
That's really awesome, thank you.
